File: 编程练习/find_max.py
def find_max(lis):
    for j in range(len(lis)):
        if j==0:
            i=lis[j]
        elif lis[j]>i:
            i=lis[j]
    return i
lis1=[0,7,9,4,5,8,3]
find_max(lis1)
'''
# 写一个函数 find_max(lst)，返回列表中的最大值。
# 
# 要求：
# 
# 不能使用内置函数 max()
# 
# 可以手写循环，逐个比较
# 
# 假设列表非空，且元素都是数字
'''

File: 编程练习/findMax.js
function findMax(numbers){
    let  max=numbers[0];
    for (let i=0;i<numbers.length;i++){
        if  (numbers[i]>=max){
            max=numbers[i];
        }
    }
    return max
}
console.log(findMax([3, 1, 4, 1, 5, 9, 2]));
console.log(findMax([-10, -5, -20]));
console.log(findMax([7]));